broad/broad/settings.py

```diff
@@ -0,0 +1,19 @@
+BOT_NAME = 'broad'
+
+SPIDER_MODULES = ['broad.spiders']
+NEWSPIDER_MODULE = 'broad.spiders'
+
+CLOSESPIDER_ITEMCOUNT = 100000
+RETRY_ENABLED = False
+COOKIES_ENABLED = True
+
+LOGSTATS_INTERVAL = 3
+LOG_LEVEL = 'INFO'
+MEMDEBUG_ENABLED = True
+CONCURRENT_REQUESTS = 120
```

Comment:
How big is the difference? The fact that some settings reduce speed is ok, we want to have a realistic broad crawl benchmark. The speed will likely drop even more with delays and errors.